sidebars.js

```diff
@@ -1,108 +1,115 @@
+        {
+            Console: [
+                'aserto-console/introduction',
+                'aserto-console/connections',
+                'aserto-console/directory',
+                'aserto-console/policies',
+                'aserto-console/view-policies',
+                'aserto-console/manage-tenant',
+            ],
+        },
+        {
+            'Authorizer Guide': [
+                'authorizer-guide/overview',
+                'authorizer-guide/authz',
+                'authorizer-guide/is',
+                'authorizer-guide/identity-context',
+                'authorizer-guide/policy-context',
+                'authorizer-guide/resource-context',
+                'authorizer-guide/query',
+                'authorizer-guide/display-state-map',
+                'authorizer-guide/decisiontree',
+                'authorizer-guide/dir',
+                'authorizer-guide/policy',
+            ],
+        },
+        {
+            SDKs: [
+                'software-development-kits/overview',
+                {
+                    type: 'category',
+                    label: 'JavaScript',
+                    collapsed: false,
+                    items: [
+                        'software-development-kits/javascript/react',
+                        'software-development-kits/javascript/spa',
+                        'software-development-kits/javascript/express',
+                    ],
+                },
+                {
+                    type: 'category',
+                    label: 'Python',
+                    collapsed: false,
+                    items: [
+                        'software-development-kits/python/api-client',
+                        'software-development-kits/python/flask',
+                        {
+                            type: 'category',
+                            label: 'Reference',
+                            collapsed: false,
+                            items: [
+                                'software-development-kits/python/identity-providers',
+                                'software-development-kits/python/authorizers',
+                            ]
+                        }
+                    ],
+                },
+                'software-development-kits/dotnetcore',
+            ],
+        },
+        {
+            CLI: [
+                'command-line-interface/introduction',
+                'command-line-interface/installation',
+                'command-line-interface/login',
+                'command-line-interface/policies',
+                'command-line-interface/connections',
+                'command-line-interface/directory',
+                'command-line-interface/onebox',
+            ],
+        },
+        {
+            'Troubleshooting': [
+                'troubleshooting/cannot-add-existing-policy',
+                'troubleshooting/leaked-secret-keys',
+                'troubleshooting/cannot-remove-github-connection',
+            ]
```

Comment:
should also add the "conflicting roots" one, while the one-runtime-per-policy isn't deployed in prod yet